security/security-spring/src/main/java/org/cbioportal/security/spring/CancerStudyPermissionEvaluator.java

```diff
@@ -108,6 +108,9 @@
 
     @Value("${filter_groups_by_appname:true}")
     private String FILTER_GROUPS_BY_APP_NAME;
+    
+    @Value("${authorization:false}")
```

Comment:
I am not sure why this class was modified.  It is only used when AUTHORIZATION=true.